ui/src/organizations/actions/orgs.ts

```diff
@@ -19,12 +19,12 @@ import {
   orgEditFailed,
   orgRenameSuccess,
   orgRenameFailed,
-} from 'src/shared/copy/v2/notifications'
+} from 'src/shared/copy/notifications'
 
 // Types
 import {Bucket} from '@influxdata/influx'
 import {Organization, RemoteDataState} from 'src/types'
-import {PublishNotificationAction} from 'src/types/actions/notifications'
+import {PublishNotificationAction} from 'src/shared/actions/notifications'
```

Comment:
Could/should this go in `src/types`? Watts centralized a bunch of types there a while ago and I found I really liked the change (don't have to remember where the various types live).